css-brochure-form/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="maindiv">
+  <h1>Learn More</h1>
+  </div>
+  <div>
+    <label for="email">Company Email</label>
+    <div>
+      <input type="email" id="email" name="email" placeholder="sally@company.co" />
+    </div>
+  </div>
+<div>
+  <label for="name">Name</label>
+  <div>
+    <input type="text" id="name" name="name" placeholder="Sally Smith" />
+  </div>
+  <div>
+    <label for="location">Where are you hiring?</label>
+    <div>
+      <input type="text" id="location" name="location" placeholder="City, Country" />
+    </div>
+  </div>
+  <div>
+    <label for="nengineer">How many engineers are you hiring in the next 6 months?</label>
+    <div>
+      <select id="nengineer" name="nengineer">
+        <option value="1-2">1-2 Engineers</option>
+        <option value="3-5">3-4 Engineers</option>
+        <option value="6-9">6-9 Engineers</option>
+        <option value="10andmore">10+</option>
+      </select>
+    </div>
+  </div>
+  <div id="submitdiv">
+    <input id="submit" type="submit">
+  </div>
```

Comment:
You should definitely put all this in a `form` element, form control elements should always be within a `form` element if users are actually submitting information with it.